provider/image.go

```diff
@@ -527,6 +533,9 @@ func (p *dockerNativeProvider) runImageBuild(
 		}
 		time.Sleep(1 * time.Second)
 	}
+	if imageID == "" {
```

Comment:
We have an `imageID == ""` on line 498 above, and not doing anything to it in between as far as I can tell.

Should a check be run in the final `findImageID` loop instead?